esrally/track/loader.py

```diff
@@ -1005,7 +1005,7 @@ def read(self, track_name, track_spec_file, mapping_dir):
             self.logger.exception("Could not load [%s].", track_spec_file)
             msg = "Could not load '{}'. The complete track has been written to '{}' for diagnosis.".format(track_spec_file, tmp.name)
             # Convert to string early on to avoid serialization errors with Jinja exceptions.
```

Comment:
Can you please remove the comment that becomes outdated by removing the `str()` call?

Reply:
yep